site/cypress/e2e/Cypress-Gradeable/office_hours_queue.spec.js

```diff
@@ -132,7 +132,86 @@ describe('test office hours queue', () => {
         editAnnouncement('');
         cy.get('[data-testid="announcement"]').should('not.exist');
         cy.get('[data-testid="popup-message"]').should('contain', 'Updated announcement');
-        // diable and delete all queue
+
+        // Confirm student queue history
+        cy.get('[data-testid="search-student-queue-input"]').first().type('student');
+        cy.get('[data-testid="search-student-queue-btn"]').first().click();
+        cy.get('[data-testid="student-row-1"]')
+            .contains('[data-testid="row-label"]', '1').siblings()
+            .contains('[data-testid="current-state"]', 'done').siblings()
+            .contains('[data-testid="queue"]', 'Lab Help').siblings()
+            // This checks if time entered and time removed are in fact times.
+            // We do not check for a specific time because this may change.
+            .contains('[data-testid="time-entered"]', /\d{4}-\d\d-\d\d \d\d:\d\d:\d\d/).siblings()
+            .contains('[data-testid="time-removed"]', /\d{4}-\d\d-\d\d \d\d:\d\d:\d\d/).siblings()
+            .contains('[data-testid="helped-by"]', 'grader').siblings()
+            .contains('[data-testid="removed-by"]', 'grader').siblings()
+            .contains('[data-testid="removal-method"]', 'helped');
+        cy.get('[data-testid="student-row-2"]').contains('[data-testid=row-label"]', '2')
```

Comment:
can you explain what exactly are you testing here? (line 150)
is you verifying that 
`cy.get('[data-testid="student-row-2"]')` should have the value 2 ??

Reply:
Yes, I'm making sure that each row of displayed table is what it's supposed to be.